Remotion/Core/Extensions/Logging/Log4Net/Log4NetLogger.cs

```diff
@@ -15,27 +15,118 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 //
 using System;
+using System.Globalization;
 using log4net.Core;
-using Microsoft.Extensions.Logging;
+using log4net.Util;
+using MicrosoftLogging = Microsoft.Extensions.Logging;
+using RemotionLogging = Remotion.Logging;
 
 namespace Remotion.Logging.Log4Net;
 
-public class Log4NetLogger
-    : Microsoft.Extensions.Logging.ILogger
+/// <summary>
+/// Implements <see cref="Microsoft.Extensions.Logging.ILogger"/> to provide an implementation with log4net.
+/// </summary>
+public class Log4NetLogger : MicrosoftLogging.ILogger
 {
-  public void Log<TState> (Microsoft.Extensions.Logging.LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
+  private readonly log4net.Core.ILogger _logger;
+
+  public Log4NetLogger (string categoryName)
+  {
+    _logger = log4net.LogManager.GetLogger(categoryName).Logger;
+  }
```

Comment:
I missed this in the first review (sorry): we need to pass the log4net logger in the ctor, not resolve it manually inside the ctor. this would create a dependency we do not want.

Reply:
We already found this locally, would've been fixed anyways :)